pkgs/development/haskell-modules/configuration-common.nix

```
@@ -2998,6 +2998,14 @@ self: super: {
   # 2024-03-25: HSH broken because of the unix-2.8.0.0 breaking change
   HSH = appendPatches [./patches/HSH-unix-openFd.patch] super.HSH;
 
+  # Support unix < 2.8 to build in older ghc than 9.6
+  linux-namespaces = appendPatch
+    (fetchpatch {
+      url = "https://github.com/redneb/hs-linux-namespaces/pull/4/commits/f4a3546541bb6c7172fdd03e177a961da60e3951.patch";
```

Githubs PR patch urls are unstable.
```suggestion
      url = "https://github.com/redneb/hs-linux-namespaces/commit/f4a3546541bb6c7172fdd03e177a961da60e3951.patch";
```